.travis.yml

```diff
@@ -22,6 +22,7 @@ before_script:
 - sbt ++$TRAVIS_SCALA_VERSION clean compile
 - echo "Wait for containers to be up and running"
 - bash waitForContainerSetup.sh
+- docker-compose exec ceph radosgw-admin user modify --uid ceph-admin --system
```

Comment:
I think we do not need this, as ceph-admin should be aready system user

Reply:
If you start a new container and you type `radosgw-admin user info --uid ceph-admin`, you can see that ceph-admin is actually not an admin (the `"system": true` entry is missing).

Reply:
ceph-admin user is beeing granted 
 radosgw-admin --cluster ceph caps add '--caps=buckets=*;users=*;usage=*;metadata=*' --uid=ceph-admin

on start, which is enough and we do not need to promote it to system.